dashboard/src/modules/components/OverviewComponent/common-component.jsx

```diff
@@ -153,3 +162,26 @@ export const RenderPagination = (props) => {
     />
   );
 };
+export const ProgressBar = (props) => {
+  const findVariant = () => {
+    if (props.percent >= 65) {
+      return null;
+    } else if (props.percent < 65 && props.percent >= 40) {
+      return ProgressVariant.warning;
+    } else {
+      return ProgressVariant.danger;
+    }
+  };
```

Comment:
At line 169, `props.percent < 65` will always be true, because if it were false then control would have entered the block at line 168 and would have never reached line 169.

Also, I wouldn't be opposed to using a compound ternary in spots like this:
```
  const findVariant = () => (
    props.percent >= 65
      ? null
      : props.percent >= 40
        ? ProgressVariant.warning
        : ProgressVariant.danger;
  );
```

Reply:
That's true. As we are not using this Progress Bar removed this component